### ISSUE TITLE
Bug or defect?

### ISSUE THREAD

**Issue by carols10cents** [laboon/ebook#25]
I'm guessing that you changed between preferring "bug" or "defect"? Section 2.4 has "bug" in its title, but that section only ever uses the word "defect" and never says "bug". Given that this is very early on in the book and you haven't defined everything yet, I think it would be better to do one of the following:

- pick one of bug or defect and only use that one
- explicitly state that there is a difference if you think there is
- explicitly state that you're going to use "bug" and "defect" interchangeably if you think there isn't a difference

**Comment by timparenti** [comment]
Some stats on `master` for consideration:

```
$ grep -irn bug text | wc -l
26
$ grep -irn defect text | wc -l
170
```

Also worth noting that "defect" has the primary definition in the glossary, while "bug" is a redirect.

**Comment by carols10cents** [comment]
Is that a 301 redirect or a 302 redirect?

**Comment by timparenti** [comment]
I was actually trying to find the actual word glossarists use for this kind of entry, but I figured "redirect" was clear in this context.

**Comment by laboon** [comment]
I mention at one point that I will prefer "defect", but use "bug" whenever I forget, as I consider them synonymous.  However, I think I will mention that they are synonymous, but use "defect" outside of that clarification.

**Comment by laboon** [comment]
Should be fixed as of commit 824e0148f6d7d73e11ec5dbf723c9745947f2514

There are actually two commits but I am lazy and didn't squash

**Comment by timparenti** [comment]
The mention comes at the end of section 9.1 (file 30, line 21):
> Finally, note that the word "bug" is often used interchangeably with the word "defect".  They mean exactly the same thing, but "bug" is much more colloquial.  In this book, the word "defect" will be used, except in those cases where author forgets to do so.
